README.md

```diff
@@ -0,0 +1,18 @@
+# Drupal Commerce Documentation
+
+## Local setup
```

Comment:
Let's encourage people to use the GitHub UI to propose changes, and have them built in a PR by Platform.sh We should entter with this info and then say "but if you want to contribute heavily, we recommend..."